src/sentry/models/group.py

```diff
@@ -716,3 +716,18 @@ def pre_save_group_default_substatus(instance, sender, *args, **kwargs):
             instance.substatus = GroupSubStatus.ONGOING
         if instance.status == GroupStatus.IGNORED and instance.substatus is None:
             instance.substatus = GroupSubStatus.UNTIL_ESCALATING
+
+        if instance.status == GroupStatus.UNRESOLVED and instance.substatus not in (
+            GroupSubStatus.ONGOING,
```

Comment:
I believe we also want to add GroupSubStatus.Regressed

Reply:
Do we also want to add `NEW` as well?

Reply:
Actually, I think I'll wait to add those in a new PR. Will also refactor the code a bit to codify the legal substatus mappings to the Group.status.